.vscode/launch.json

```diff
@@ -14,7 +14,7 @@
       },
       "runtimeArgs": [
         "-p",
```

Comment:
Maybe change commit summary to use "vscode" as "vs[space]code" does not seem common